client/blocks/reader-full-post/index.jsx

```diff
@@ -243,6 +238,22 @@ export class FullPostView extends React.Component {
 		}
 	}
 
+	goToNextPost = () => {
+		if ( getLastStore() ) {
```

Comment:
Could we not just set `const lastStore = getLastStore();` at the top of the `goToNextPost` method to avoid repeating the function call?